src/main/java/com/glodblock/github/inventory/MEMonitorIFluidHandler.java

```diff
@@ -189,6 +191,22 @@ public IItemList<IAEFluidStack> getAvailableItems(IItemList out) {
         return out;
     }
 
+    @Override
+    public IAEFluidStack getAvailableItem(@Nonnull IAEFluidStack request) {
+        long count = 0;
+        for (final IAEFluidStack is : this.cache) {
```

Comment:
Would it be faster to use `findPrecise` in this instance (which corresponds to a hashmap lookup)

Reply:
Done, I was blindly copying from getAvailableItems :sweat_smile: